### DISCUSSION TITLE
Feature Request: Smart Search (finder) plugin

### DISCUSSION THREAD

**Discussion by jojo12** [slabbi/uddeIM#97] [Ideas]
With Joomla!4+ ended com_search. If you update from Joomla!3 you still have search plugins and they work. 
If you install Joomla 4+ new, then you don't get com_search!
Because Joomla says, that finder plugins are smarter and more intelligent than search plugins. So the question is: Is it possible to create a uddeIM smart search plugin? Make this sense? And how to do this?

**Comment by joomod** [comment]
?? hmm,
I don't know how the uddeim_searchbot.php exactly works.
hope slabbi can give an answer

**Comment by jojo12** [comment]
I think, it is a search plugin not a finder plugin. Am I right @slabbi?

**Comment by slabbi** [comment]
Correct. Please don't ask me about the Joomla finder. My last coding for Joomla was for J4 beta. I am using Wordpress for about eight years now.

**Comment by joomod** [comment]
I'm trying to find out for finder

**Comment by joomod** [comment]
so after it seems we have a very "issue-free" version I have tried and looked for a finder plugin,
BUT
as far as I understand the com_finder extension it is not usefull for a PM system.

**Finder** indexes all content to be faster, but to keep privacy on a PM system this index must then
be created **for each user** seperately, as a search should only be available for the user's personal pms.

so I decided it is more useful to **keep** the old searchbot and include the basic **com_search** and **mod_search**
(availble for J4, - two error corrections for J5) to an **uddeim_search package** along with the **plug_uddeim_searchbot**

original:  https://github.com/joomla-extensions/search/releases/

as I noticed during tests:
some extensions (e.g. weblinks) install still a com_search plugin, but are not J5 compatible,
so make shure **other com_search plugins are disabled**.

you see in the package: com+mod+plug search are only 40kB.

[pkg_uddeim_search_j50.zip](https://github.com/user-attachments/files/16325270/pkg_uddeim_search_j50.zip)

![udd_search](https://github.com/user-attachments/assets/5088b571-7507-4063-a7c6-045256de21cd)
